opm/simulators/linalg/PropertyTree.hpp

```diff
@@ -20,51 +20,136 @@
 #ifndef OPM_PROPERTYTREE_HEADER_INCLUDED
 #define OPM_PROPERTYTREE_HEADER_INCLUDED
 
+#include <functional>
```

Comment:
probably missing something but i don't see where this is required?

Reply:
> probably missing something but i don't see where this is required?

We use [`std::less<>`](https://en.cppreference.com/w/cpp/utility/functional/less) in the [`ptree` definition](https://github.com/OPM/opm-simulators/blob/f73a027edc116a407224b1d911d2b7caf7f55550/opm/simulators/linalg/PropertyTree.hpp#L31).